tests/examples.js

```diff
@@ -441,6 +441,50 @@ function throttleTest(end) {
 
 }
 
+function debaunceTest(end) {
+  let state;
+
+  function fn(letter) {
+    console.log('Debaunced function, state: ' + state);
+    if (state === letter) {
+      console.log('Debaunce test done');
+      end();
+    }
+  }
+
+  const f1 = metasync.debaunce(500, fn, ['I']);
+
+  // to be called one time (E)
+  state = 'A';
+  f1();
+  state = 'B';
+  f1();
+  state = 'C';
+  f1();
+  state = 'D';
+  f1();
+  state = 'E';
+  f1();
+
+  // to be called 2 times (last will be I)
```

Comment:
Oh, sorry. I is a mistake. Overall result is ok (`E + I`). I will change comment though.